client/components/index.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * External Dependencies
+ */
+// Turn on react-dates classes/styles, see https://github.com/airbnb/react-dates#initialize
+import 'react-dates/initialize';
+
+// Simple components
+export { default as Card } from './card';
+export { default as Count } from './count';
+export { default as DropdownButton } from './dropdown-button';
+export { default as Flag } from './flag';
+export { default as Gravatar } from './gravatar';
+export { default as Link } from './link';
+export { default as OrderStatus } from './order-status';
+export { default as Pagination } from './pagination';
+export { default as ProductImage } from './product-image';
+export { default as SegmentedSelection } from './segmented-selection';
+export { default as SplitButton } from './split-button';
+
+// Complex components
```

Comment:
Complex components are ones that require more data-driven props like a product object? Is this akin to blocks vs components in Calypso?

Reply:
The complex components were more likely to be made up of other components, or export multiple components (like `TableCard` or `EllipsisMenu`), but ultimately I couldn't make a great distinction between the two, so I just left it like this. We can merge all this back into a single alphabetized list, or try to formalize a difference. This might get more complicated as we get components that also use api requests.

Reply:
I'd vote for just keeping it simple and alphabetizing a single list. It'll be easy to add a new one without having to think about which group it goes under.